prettyPrinters/rust_types.py

```diff
@@ -37,6 +42,11 @@ class RustType(object):
 STD_STRING_REGEX = re.compile(r"^(alloc::([a-z_]+::)+)String$")
 STD_STR_REGEX = re.compile(r"^&str$")
 STD_OS_STRING_REGEX = re.compile(r"^(std::ffi::([a-z_]+::)+)OsString$")
+STD_OS_STR_REGEX = re.compile(r"^((&|&mut )?std::ffi::([a-z_]+::)+)OsStr( \*)?$")
+STD_PATH_BUF_REGEX = re.compile(r"^(std::([a-z_]+::)+)PathBuf$")
+STD_PATH_REGEX = re.compile(r"^(&?std::([a-z_]+::)+)Path( \*)?$")
+STD_CSTRING_REGEX = re.compile(r"^(std::ffi::([a-z_]+::)+)CString$")
+STD_CSTR_REGEX = re.compile(r"^(&?std::ffi::([a-z_]+::)+)CStr( \*)?$")
```

Comment:
Do I understand correctly that we don't suppose `&mut CStr` because it's quite hard to get mutable reference to `CStr`?

Reply:
Yes. To double-check, I googled a little and found a [comment](https://github.com/rust-lang/rust/issues/54063#issuecomment-419657173) saying that `CStr` is not intended to be mutable